exercises/isbn-verifier/canonical-data.json

```diff
@@ -149,6 +149,15 @@
             },
             "expected": false
         },
+        {
+            "uuid": "6ec400c2-6421-11ec-97c5-8c1645feb539",
+            "description": "invalid characters are not ignored in valid isbn",
+            "property": "isValid",
+            "input": {
+                "isbn": "3598P215088"
+            },
+            "expected": false
+        },
```

Comment:
I like this test case but I find it a bit difficult to differentiate from the one above. I am thinking whether the best course of action might be to call this one `"invalid characters are not ignored"` and make it a [re-implementation](https://github.com/exercism/problem-specifications#changing-tests) of the one above.

There is no way to tell with the test whether the test case above (line 144) actually failed because of the invalid character or just because the isbn was invalid. Because of that, the new test case would cover that "not ignoring" much better.

Reply:
Yes, a single test makes more sense. Updated accordingly.